src/BUILD

```diff
@@ -311,16 +316,19 @@ cc_binary(
 cc_binary(
     name = "libcustom_node_add_one.so",
     srcs = [
-        "custom_nodes/add_one/add_one.cpp",
-        "custom_node_interface.h",
-        "queue.hpp",
+        "custom_nodes/common/utils.hpp",
         "custom_nodes/common/buffersqueue.hpp",
         "custom_nodes/common/buffersqueue.cpp",
         "custom_nodes/common/customNodeLibraryInternalManager.hpp",
         "custom_nodes/common/customNodeLibraryInternalManager.cpp",
-        "custom_nodes/common/utils.hpp",
+        "queue.hpp",
+        "custom_nodes/add_one/add_one.cpp",
+        "custom_node_interface.h",
         "custom_nodes/add_one/addOneInternalManager.hpp"
     ],
+    deps = [
+        "@opencv//:opencv"
```

Comment:
We should not include opencv to compile this custom node. This node does not use opencv at all.